fastparquet/test/test_dataframe.py

```diff
@@ -91,3 +93,14 @@ def test_timestamps():
                                                                   't2': z2})
     assert df.t1.dt.tz.zone == z
     assert df.t2.dt.tz.zone == z2
+
+
+def test_pandas_hive_serialization():
+    column = "data"
+    df = pd.DataFrame(
+        columns=[column], data=[("42",), ("",), ("0",), ("1",), ("0.0",)]
+    )
+    df.to_parquet("test.par", file_scheme="hive", row_group_offsets=[0, 2, 4])
```

Comment:
Would be better with a temporary directory (`tempfile.mkdtemp` or `tmpfile` fixture), since the local directory may not be writable.

Reply:
Thanks for that hint 🙂 That will make a lot of our internal tests much more beautiful 👍 Adapted this test to it, too 🙂